src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -504,11 +380,18 @@ public void SetOptions(DacDeployOptions options)
         public void SetGenericDeployOptionProps(PropertyInfo prop, DacDeployOptions options, PropertyInfo deployOptionsProp)
         {
             var val = prop.GetValue(options);
-            var attribute = prop.GetCustomAttributes<DescriptionAttribute>(true).FirstOrDefault();
+            var descriptionAttribute = prop.GetCustomAttributes<DescriptionAttribute>(true).FirstOrDefault();
+            var displayNameAttribute = prop.GetCustomAttributes<DisplayNameAttribute>(true).FirstOrDefault();
```

Comment:
what is the true getting passed in here? Can you add the parameter name?

Reply:
It doesn't matter what we pass here true/false/nothing, but cannot pass a string type. Will make it empty.

prop.GetCustomAttributes<DescriptionAttribute>().FirstOrDefault()